src/GEM_u8g2.cpp

```diff
@@ -295,18 +308,18 @@ void GEM_u8g2::printMenuItemValue(GEMItem* menuItemTmp, byte yDraw)
   switch (menuItemTmp->linkedType) {
     case GEM_VAL_INTEGER:
         itoa(*(int*)menuItemTmp->linkedVariable, valueStringTmp, 10);
-        printMenuItemValue(valueStringTmp);
+        printMenuItemValuePrintFunction(valueStringTmp, yText,(menuItemTmp->title == nullptr));
```

Comment:
Puhh you do not have to forget to set the last parameter of this function to a nullptr. Otherwise you are in a loop ^^ 
Should this mentioned in any comment? What do you think?